pkg/kn/commands/service/list_mock_test.go

```diff
@@ -69,7 +69,7 @@ func TestServiceListAllNamespaceMock(t *testing.T) {
 	assert.NilError(t, err)
 
 	outputLines := strings.Split(output, "\n")
-	assert.Assert(t, util.ContainsAll(outputLines[0], "NAMESPACE", "NAME", "URL", "GENERATION", "AGE", "CONDITIONS", "READY", "REASON"))
+	assert.Assert(t, util.ContainsAll(outputLines[0], "NAMESPACE", "NAME", "URL", "LATESTCREATED", "LATESTREADY", "AGE", "CONDITIONS", "READY", "REASON"))
```

Comment:
Would these two words read better was `LATEST CREATED` and `LATEST READY` assuming that's not what's exactly coming from APIs?

Reply:
Change to `LATEST REVISION`